components/the-lab-membership.vue

```diff
@@ -0,0 +1,126 @@
+<script setup>
+import { ref, onMounted } from 'vue'
+import { tiers, options } from '~/utilities/membershipsData.js'
+import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
+
+const theTiers = ref(null)
+const theOptions = ref(null)
+const contentIndex = ref(0)
+
+onMounted(() => {
+  theTiers.value = tiers
+  theOptions.value = options
+})
+
+/**
+ * Function connected to the emitter from the Carousel component when the content has changed, it provides the index of the page showing 
+ */
+const emitting = (e) => {
```

Comment:
I'm not experienced with vue, but it _looks_ like this is setting the index of the carousel on page refresh to start the carousel where the user already was (instead of restarting at index 0). If that's the case, `emitting` isn't a great function name (what is being emitted? what does that mean?). I'd consider a name more like `setCarouselStartIndex` or something like that.

Reply:
no, the Carousel comp has an emit in it called "update:page" which returns the page it is currently on. I am able to bind that internal emit to a function, which I am calling "emitting" to update a variable for other elements to react to. I'll change the name of the method to be more informative.

const carouselEmitCurrPage = (pageNum) => {
  contentIndex.value = pageNum
}